scripts/fiber/tests-failing.txt

```diff
@@ -87,6 +72,9 @@ src/renderers/art/__tests__/ReactART-test.js
 * resolves refs before componentDidMount
 * resolves refs before componentDidUpdate
 
+src/renderers/dom/__tests__/ReactDOMProduction-test.js
+* should throw with an error code in production
```

Comment:
Hm. This is strange that this was passing before. This is probably a behavior change that we want to allow.